### DISCUSSION TITLE
Standard list view

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#292] [Ideas]
Hi @razonyang.

I know that this is a cards theme, but will you consider to introduce a standard list view like we had on your previous theme [HBS](https://hbs.razonyang.com/v1/en/)?

![HBS](https://github.com/hbstack/theme-cards/assets/49299328/54a3924e-c21e-4a64-98fc-5b276b215e11)

I prefer how it looked.

I tried to create it with this theme by changing the list_cols_md parameter to 1, but then it feels strange, because the featured image gets stretched.

![Cards-theme](https://github.com/hbstack/theme-cards/assets/49299328/c5e7ab01-cfa3-4ab1-9b46-1c8956f2fb1a)

Thanks!

**Comment by razonyang** [comment]
Hmm, it will make maintainence harder, you can create a feature request, but it won't happen fast (no ETA).

Btw, you can increase the image height if you want one post per row, for example.

```scss
.hb-blog-post-card-img {
    height: 320px;
}
```

**Reply by LITUATUI** [reply]
Thanks, no hurry.
